### ISSUE TITLE
Make path split function more prominent

### ISSUE THREAD

**Issue by Echolon** [westnordost/StreetComplete#2309]
Hello,

I just found out there is the awesome path split functions! What a pity I found it just today because it makes so much sense!

Therefore I would like to make it way more prominent and not hide it in a sub-menu.

**Use case**
Mappers will consider this option and improve paths to their physical reality.

**Proposed Solution**
Offer a `First split path` button ahead of the path surface options

**Comment by westnordost** [comment]
Glad you like it, it is quite complex technically so that it can be very easy for the users.

The option to split the way first is available for a range of quests. It would not be a consistent UI if for one quest (surface quest), the option is presented at a different point. What did you do before you found that option? I would have expected that users would then not answer the question at all or press "can't say" and attempt to describe the situation.

**Comment by Helium314** [comment]
For the new surface quest, one needs to scroll down across all available surfaces. It isn't an enourmous amount of work, but still I found having to scroll that far to the `can't say` menu can be bit annoying.
So that's maybe why the proposal was to change it for the surface quest.

**Comment by Echolon** [comment]
> I would have expected that users would then not answer the question at all or press "can't say" and attempt to describe the situation.

Either that, especially if the paths cannot be created as a single element and may cross several stress or corners. Or I just took the 90% marjority surface it if only small parts of a path piece different.

**Comment by matkoniecz** [comment]
> Its really hidden, because 98% of the cases it simple to answer

What you tried in this 2% of cases before noticing split way option?

There is no bad answer here, I am asking here as info what actual users are doing is valuable.

(Though it is not possible to handle all possible user behaviors).

**Comment by mnalis** [comment]
> Either that, especially if the paths cannot be created as a single element and may cross several stress or corners. Or I just took the 90% marjority surface it if only small parts of a path piece different.

Is that very small part different enough to be important? I'd argue that often it is not (for example, "this asphalt footway has a 1 meter of concrete" in it is actually harmful). Sure, it might be easy to say "_more details is always better_", but it does come at the _very real expense_:

- One is that one simple split now by one person means extra work for lots of people forever. Let's say one person added that one meter on concrete on otherwise asphalt footway and map is now little more precise. But now **everyone else**  for **infinity** have double work to do - if I want to add `lit` status, I have two quest to answer instead of one. If someone else wants to add it in some relation, it has to select and add two segments, etc. Now imagine if that 100m way is split 20 times. Now imagine hundreds of millions of such ways. Ouch.

- Also, while official OSM stand might still be "just add more detail", huge amounts of data is starting to rear its ugly head: just making a database dump on powerful servers takes almost 4 days now, [distributing a planet has to be throttled](https://planet.openstreetmap.org/) until we [move to bittorrent](https://github.com/openstreetmap/operations/issues/451) as it overloads the network and slows it down, updating the tiles and other maps related databases takes longer and longer time so you're [getting stale maps and services become unusable](https://github.com/westnordost/StreetComplete/issues/1700) etc. 

TL;DR: what I'd argue for is to let `split way` functionality be a little harder to get at (as it is now), so people won't overly abuse it.

**Comment by Echolon** [comment]
Interesting point. However, it should be known that it is available. I mean... otherwise they might just tag the path wrongly and in my case Id like to split up weird setups mostly not each meter of 100m path.

**Comment by westnordost** [comment]
Well, I'll close this as will not fix. @mnalis brought up some arguments, but for me, it is most important that the UI is consistent. I cannot add this function somewhere in the "normal form area" in this quest but not in others.
